invenio_theme/webpack.py

```diff
@@ -71,7 +71,7 @@
                 '@js/invenio_theme': 'js/invenio_theme',
                 # TODO: change to "@less/invenio_theme":
                 #   https://github.com/inveniosoftware/invenio-theme/issues/200
-                '@invenio_theme/less': 'less/invenio_theme',
+                '@less/invenio_theme': 'less/invenio_theme',
```

Comment:
I might be wrong, but the issues are not going to pop up in this module by changing the aliases. But in other modules that are trying to import and use this. Consider the case that in my own invenio module I import some styles from `invenio-theme` (again not sure if this is a case, or how to check them)

`@import '@less/invenio_theme/theme/elements/button.variables';`

Reply:
You are correct on this. I did a quick check to see if it happens, and did not find anything. Plus the fact that both cookiecutters display correctly (and there were no errors building the assets) means that all modules involved in them (the ones we actually care about in this PR) are Ok.